frontend/src/components/Grades/GradesTable.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { getGrades } from '../../storage/gradesTable';
+import { getActivityTypeName, percentagesToGrade } from '../../utils/constants';
+import { TableContainer } from '../PointsPage/Table/TableStyle';
+
+export default function GradesTable() {
+    const tableContent = getGrades();
+
+    return (
+        <TableContainer striped bordered hover>
+            <thead>
+                <tr>
+                    <th>Nr</th>
+                    <th>Aktywność</th>
+                    <th>Ocena</th>
+                    <th>Punkty</th>
+                    <th>Data</th>
+                    <th>Typ aktywności</th>
+                </tr>
+            </thead>
+            <tbody>
+                {tableContent.map((row, idx) => (
+                    <tr key={idx}>
+                        <td>{idx+1}</td>
+                        <td>{row.activityName}</td>
+                        <td>{percentagesToGrade(row.points / row.maxPoints)}</td>
+                        <td>{row.points}/{row.maxPoints}</td>
+                        <td>{row.date}</td>
+                        <td>{getActivityTypeName(row.activityType)}</td>
```

Comment:
To też bym wyciągnął jak wyżej. Dodatkowo bardzo podobnie wygląda to w `UnmarkedActivitiesTable.js`. Jedyny pomysł jaki mam w tej chwili jak to zrobić jest następujący:
```js
const TableType = {
    GRADES_TABLE: 0,
    UNMARKED_ACTIVITIES_TABLE: 1
};

const TableContent = (tableType, row, idx) => {
    id: idx + 1,
    name: row.activityName,
    points: tableType === TableType.GRADES_TABLE 
        ? percentagesToGrade(row.points / row.maxPoints) 
        : row.maxPoints,
    date: row.date,
    activityType: getActivityTypeName(row.activityType)
}
```
No i wtedy można analogicznie rozpisać mapper na `<tr>`. To taki pomysł na szybko, w dodatku nietestowany. Jeżeli ktoś wymyśli coś lepszego to zapraszam do dyskusji 😉

Reply:
W sumie możesz też wykorzystać moment.js do wyświetlania dat. Gdy zamiast `row.date` napiszesz `moment(row.date).calendar()` to data, która zawiera godzinę może się wyświetlać np tak: "Dziś o 17:40", albo "Wczoraj o 13:25" itp. I prawdopodobnie jakby nie było godzin to takie daty bliskie "dzisiejszej" wyświetlały by się w stylu: "Dzisiaj", "Wczoraj". 
Tutaj dodatkowa uwaga: domyślnie wyświetla się data po angielsku. Żeby mieć polski wystarczy zapisać: 
 ```js
import moment from "moment";  // żeby po prostu skorzystać z moment.js
import 'moment/locale/pl';  // żeby zmienić na j.polski
 ```
 Jest to oczywiście tylko bajer/wodotrysk. Możesz to pominąć lub dodać według uznania 😉